### DISCUSSION TITLE
[react-markdown] ie11 support

### DISCUSSION THREAD

**Discussion by R-Gopi** [remarkjs/remark#707] [Q&A] [answered]
React markdown viewer not working in ie11,how to solve this tried by adding poly fills but no luck

**Comment by ChristianMurphy** [comment, marked as answer]
React Markdown uses both new syntax features (like `const`) and new APIs (like `Object.assign`).
Polyfills work for new APIs, for new syntax a transpiler is needed.
Consider using babel preset env https://babeljs.io/docs/en/babel-preset-env and set it to support your desired browser matrix.

**Reply by vijayapalemis** [reply]
Already i am using the react-app-polyfill to support the above features, but still i am getting syntax error
![image](https://user-images.githubusercontent.com/45386995/117661119-98790200-b1bb-11eb-8d70-87f0864ab35d.png)

**Reply by wooorm** [reply]
You need to configure the tools you are using

**Reply by ChristianMurphy** [reply]
> i am using the react-app-polyfill to support the above features, but still i am getting syntax error

@vijayapalemis as noted in the original response.
You need both polyfills, and a transpiler. `react-app-polyfill` is a polyfill but not a transpiler, you still need a traspiler like babel, or an equivalent tool.
If you are using create react app, make sure you are using the latest version of react scripts, and follow the guide on legacy browser support https://create-react-app.dev/docs/supported-browsers-features.
If you have a custom webpack/build system, you'll need to add the transpiler manually.
